swagger.yaml

```diff
@@ -3105,6 +3105,19 @@ definitions:
               format: uuid
             hostname:
               type: string
+      selected-installation-disk-paths:
```

Comment:
```suggestion
      selected_installation_disk_paths:
```

Reply:
The - delimiter is used frequently in the swagger file.There is no need to change it to _